pandas/core/sorting.py

```diff
@@ -319,6 +319,33 @@ def nargsort(
     return indexer
 
 
+def nargminmax(values, method: str):
```

Comment:
i don’t think these
should be in base.py 

rather in array_ops

Reply:
The reason I put them here is because I think it makes sense to keep it close to `nargsort`, since the code is very similar (using the same approach with the `idx`/`non_nan_idx` etc)

Reply:
> i don’t think these should be in base.py

BTW, this is not `base.py`, but `core/sorting.py`, which groups a whole bunch of functionality related to sortable values. 
(it might make sense to move `sorting.py` into the `array_ops` submodule, but I would do that as a separate move)

Reply:
k that makes sense (and let’s move sorting.py) as a followon